src/components/GenreSelect/GenreSelect.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from "react";
+import "./styles.css";
+
+const defaultSelected = "All";
+
+class GenreSelect extends Component {
+  state = {
+    genres: [],
+    selected: "",
+    onSelect: (genre) => {},
+  };
+
+  constructor(props) {
+    super(props);
+    this.state.genres = Array.from(props.genres) || new Array() < "" > 0;
+    this.state.selected = props.selected || defaultSelected;
+    console.log(this.state.selected);
+    this.state.onSelect = props.onSelect;
+  }
+
+  select = (genreName) => {
+    this.state.onSelect(genreName);
+  };
+
+  render() {
```

Comment:
When you click on a specific genre, the section is not highlighted. I guess you need to store the state of the selected genre in the parent component and change it on onSelect.

https://www.pluralsight.com/guides/how-to-handle-communication-between-parent-and-child-components-in-reactjs

Reply:
Fixed (61a6a37713a95381789739439d3619d433b7c6ba)